server/db/driver/pg/pg.go

```diff
@@ -0,0 +1,121 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package pg
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/decred/dcrdex/server/market/types"
+)
+
+// type Driver struct{}
+
+// func (d *Driver) Open(ctx context.Context, cfg interface{}) (db.DEXArchivist, error) {
+// 	switch cfg.(type) {
+// 	case *Config:
+// 		return NewArchiver(ctx, cfg)
+// 	case Config:
+// 		return NewArchiver(ctx, &cfg)
+// 	default:
+// 		return nil, fmt.Errorf("invalid config type %t", cfg)
+// 	}
+// }
+
+// func init() {
+// 	db.Register("pg", &Driver{})
+// }
+
+const (
+	defaultQueryTimeout = 20 * time.Minute
+)
+
+// Config holds the Archiver's configuration.
+type Config struct {
+	Host, Port, User, Pass, DBName string
+	HidePGConfig                   bool
+	QueryTimeout                   time.Duration
+
+	// MarketCfg specifies all of the markets that the Archiver should prepare.
+	MarketCfg     []*types.MarketInfo
+	CheckedStores bool
```

Comment:
Right, will document.  Although I'm not at all certain this is a setting we'd want to retain into production.  If the consumer of the DB is operating correctly, the checks are not needed.  But during development (perhaps beyond), it can be useful for identifying bugs.